appservice/src/startStreamingLogs.ts

```diff
@@ -55,21 +56,24 @@ export async function startStreamingLogs(client: ISimplifiedSiteClient, verifyLo
                     timerId = setInterval(async () => await pingFunctionApp(client), 60 * 1000);
                 }
 
-                await new Promise<void>((onLogStreamEnded: () => void, reject: (err: Error) => void): void => {
-                    const logsRequest: request.Request = request(`${client.kuduUrl}/api/logstream/${logsPath}`, {
-                        auth: {
-                            user: creds.publishingUserName,
-                            password: nonNullProp(creds, 'publishingPassword')
-                        },
-                        headers: {
-                            'User-Agent': appendExtensionUserAgent()
-                        }
-                    });
+                const buffer: Buffer = Buffer.from(`${creds.publishingUserName}${nonNullProp(creds, 'publishingPassword')}`);
+                const abortController: AbortController = new AbortController();
 
+                const logsRequest: Request = new Request(`${client.kuduUrl}/api/logstream/${logsPath}`, {
```

Comment:
Maybe use @azure/core-http instead of node-fetch directly?

Reply:
Gonna use @azure/ms-rest-js which is already used in several other places in this project.